configs/yolo/yolov3_ms_aug_273e.py

```diff
@@ -0,0 +1,131 @@
+# Copyright (c) 2019 Western Digital Corporation or its affiliates.
+
+# model settings
+model = dict(
```

Comment:
Please follow the [name style](https://github.com/open-mmlab/mmdetection/blob/master/docs/config.md#config-name-style) of config file.

Reply:
Sure.